packages/zowe-explorer/src/globals.ts

```diff
@@ -99,6 +100,7 @@ export let ACTIVATED = false;
 export let PROFILE_SECURITY: string | boolean = ZOWE_CLI_SCM;
 export let SAVED_PROFILE_CONTENTS = new Uint8Array();
 export const JOBS_MAX_PREFIX = 8;
+export const OPEN_DIALOG_DEFAULTURI = vscode.workspace.workspaceFolders?.[0]?.uri ?? vscode.Uri.file(os.homedir());
```

Comment:
Should we add a utility function that returns this value? I think the value of `vscode.workspace.workspaceFolders` could change if a workspace is opened, closed, or modified during a VS Code session. So we may want to check the latest value instead of caching it at launch time.

Reply:
addressed in commit https://github.com/zowe/vscode-extension-for-zowe/pull/2204/commits/8930a2f2c6a175146d5046f2e3dbe3604c0649cd